language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -327,7 +327,7 @@ tGetLedgerConfiguration withSandbox = testCase "tGetLedgerConfiguration" $ run w
     xs <- getLedgerConfiguration lid
     Just (Right config) <- liftIO $ timeout 1 (takeStream xs)
     let expected = LedgerConfiguration {
-            maxDeduplicationTime = Duration {durationSeconds = 86400, durationNanos = 0}}
+            maxDeduplicationTime = Duration {durationSeconds = 1800, durationNanos = 0}}
```

Comment:
@tudor-da I assume this is expected?

Reply:
Yes. This is an optimization for Sandbox-on-X. For simplicity, deduplication is done in-memory for Sandbox-on-X. For that reason, the default maxDeduplicationTime is 30 minutes with the maximum configurable to one hour